images/macos/toolsets/toolset-13.json

```diff
@@ -112,15 +91,14 @@
             "cocoapods",
             "xcpretty",
             "bundler",
-            "fastlane",
-            "jazzy"
+            "fastlane"
         ]
     },
     "go": {
-        "default": "1.17"
+        "default": "1.18"
     },
     "node": {
-        "default": "16",
+        "default": "18",
         "nvm_versions": [
             "14",
```

Comment:
Should we remove `nvm_versions` since we don't install nvm to images?

Reply:
The same for `mongodb` and `postgresql`

Reply:
I believe it is ok to remove packages from the list that will not be installed.